lib/generators/shopify_app/home_controller/home_controller_generator.rb

```diff
@@ -18,7 +18,11 @@ def create_products_controller
       end
 
       def create_home_index_view
-        template("index.html.erb", "app/views/home/index.html.erb")
+        if Rails.version.split(".").first.to_i >= 7
+          template("importmap_index.html.erb", "app/views/home/index.html.erb")
+        else
+          template("index.html.erb", "app/views/home/index.html.erb")
+        end
```

Comment:
This solution only works for Rails 7 so we needed to use the old template for previous versions